[Bug]: In the Classification store view the X button deletes without warning

mcop1 opened the pull request:
Resolves #11754

mcop1 committed:
[Bug]: In the Classification store view the X button deletes without warning

kingjia90 reviewed (changes requested):
the alert is fine, but the change is in the wrong file/datatype, it was supposed to be in the classificationstore (deleteGroup), not in the fieldcollection. 

IMHO The message is a little too long and "obvious", i would remove the part "After the next save all values from this group will be empty." 🤔

mcop1 commented:
> the alert is fine, but the change is in the wrong file/datatype, it was supposed to be in the classificationstore (deleteGroup), not in the fieldcollection.
> 
> IMHO The message is a little too long and "obvious", i would remove the part "After the next save all values from this group will be empty." 🤔

You are right, I added the message for the deleteGroup method too and changed the label to something more generic. Regarding the text I think the message should communicate that the whole group gets deleted, not only a single value.

mcop1 committed:
Added error message for cstore

mcop1 committed:
Changed error message
